platform/mds/mds-test-bundle/src/main/java/org/motechproject/mds/test/domain/RevertFromTrash.java

```diff
@@ -0,0 +1,32 @@
+package org.motechproject.mds.test.domain;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+@Entity(recordHistory = true)
+public class RevertFromTrash {
```

Comment:
change the name of test class. This class should be in package domain.revertFromTrash, however the name should be normal, look for the entity names in other tests